wg/multiple-dc-enhancemant/README.md

```diff
@@ -8,6 +8,10 @@ TiDB across multiple data centers.
 List the stakeholder SIGs of this Working Group.
 
 ## Meetings
+* 2019-12-03
+  > Send an issue to etcd to show proto changes.
+  > Progresses of delegated peers are maintained on delegates.
```

Comment:
what do you mean by "are maintained on delegates"?

Reply:
Something like "progresses are managed on delegates". Please help me to improve it.